src/gennylib/test/PhaseLoop_test.cpp

```diff
@@ -380,26 +376,6 @@ TEST_CASE("Actual Actor Example") {
                     {94, 3}});
     }
 
-    SECTION("Throws with other keywords in Nop phases") {
```

Comment:
My intention with deleting this test is to defer YAML validation to the "strict mode" ticket.